grakn-engine/src/main/java/ai/grakn/engine/cache/EngineCacheStandAlone.java

```diff
@@ -140,5 +143,23 @@ public void clearAllJobs(String keyspace){
         updateLastTimeJobAdded();
         if(castings.containsKey(keyspace)) castings.remove(keyspace);
         if(resources.containsKey(keyspace)) resources.remove(keyspace);
+        if(instanceCounts.containsKey(keyspace)) instanceCounts.remove(keyspace);
+    }
+
+    //-------------------- Instance Count Jobs
+
+    @Override
+    public Map<TypeLabel, Long> getInstanceCountJobs(String keyspace) {
+        return instanceCounts.computeIfAbsent(keyspace, (key) -> new ConcurrentHashMap<>());
```

Comment:
Might be better to use `getOrDefault`, to avoid putting unnecessary empty stuff in the `Map`.